packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -493,10 +506,22 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
                                 'manage',
                                 'SavedChart',
                             ) && (
-                                <LinkMenuItem
+                                <MenuItem2
```

Comment:
Would it work if we kept the `LinkMenuItem` with `href` + `onClick`? So people could still right click the menu item to edit charts (especially charts that are not specific to the dashboard ) in other tabs.

Reply:
I'll give it a shot

Reply:
Yup, looks good. Done.